bpo-40890: Add `mapping` attribute to dict views (#20691)
python/cpython: Doc/library/stdtypes.rst

   .. versionchanged:: 3.8
      Dictionary views are now reversible.

.. describe:: dictview.mapping

>    Return the dictionary from which the view was derived.  If ``d`` is a

Reviewer: ```suggestion
   Return the dictionary the view refers to.  If ``d`` is a
```
